client/src/components/modal-forms/ReturnOfServiceForm.js

```diff
@@ -126,6 +169,25 @@ export const ReturnOfServiceForm = ({
                 ]}
                 boldLabel={true}
               />
+              {values.sameSite === false && (
+                <Box mt={3} mb={2}>
+                  <Field
+                    name='site'
+                    component={RenderSelectField}
+                    label='Select Site'
+                    options={mapToOptions(sites)}
+                    onChange={(event) => handleSiteSelection(event, setFieldValue)}
+                  />
+                  <br />
+                  <Field
+                    name='ha'
+                    component={RenderTextField}
+                    label='Health Authority'
+                    disabled={true}
```

Comment:
Do we need to display this field if it's always disabled?

Reply:
This is as per the design